frontend/src/components/worksheets/WorksheetItemList.js

```diff
@@ -304,7 +303,7 @@ class WorksheetItemList extends React.Component {
                                 ],
                                 header: ['field', 'generalized-path', 'post-processor'],
                                 schema_name: '',
-                                sort_keys: [-1],
+                                sort_keys: [0],
```

Comment:
```suggestion
                                sort_keys: [-1],
```

Reply:
I think the sort_keys should still be [0]. Here it is the sort_keys, not the after_sort_key. All the sort_keys (except for the dummy item) should be >=0

Reply:
Oh, I see, thanks for pointing that out.